### PR TITLE
Issue/5581 more menu dark mode

### LINKED ISSUE #5581: Dark mode for "More" screen.
Following the conversation here https://github.com/woocommerce/woocommerce-android/pull/5399#pullrequestreview-822565173 , let's work on the Dark mode for this screen once the design is created.

### PR DESCRIPTION
**Pull request by JorgeMucientes**
Closes: #5581 

### Description
Couple things in this PR: 
- [x] Fixed dark mode colors for More menu following designs vbzgTKHV9XsyyN5cxxAuhv-fi-2496%3A90145
- [x] Removed toolbar from More menu

| Before  | After |
| ------------- | ------------- |
|   |   |

### Testing instructions
Just open more menu and check UI is displayed as in the "After" screenshot

### CODE REVIEW

File: WooCommerce/src/main/res/values-night/colors_base.xml [hunk 1]
```diff
@@ -126,4 +126,10 @@
     <color name="reader_update_progress_color">@color/woo_purple_40</color>
     <color name="reader_update_rest_color">@color/woo_purple_20</color>
     <color name="reader_update_border_color">#7F54B3</color>
+
+    <!--
+    More menu button
+    -->
+    <color name="more_menu_button_background">#232323</color>
+    <color name="more_menu_button_icon_background">#383838</color>
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/res/values-night/colors_base.xml:134]: Not the biggest fan of hardcoding colors here. I've rised some concerns about it in Slack to try to address this kind of situations in a consistent way. p1643817925982729/1643040273.030100-slack-C02N7N92DEV

**Review by hafizrahman — APPROVED** [review]
This looks good to me! Thanks for working on this and about removing the top bar as well.

Do we want to merge now or sync with iOS first?

**Comment by hafizrahman** [comment]
After some Slack discussion, we agree to merge this first.

I've fixed the merge conflict (and fixed the post-merge conflict issue 😅) and added the notification badge to the preview.

**Comment by JorgeMucientes** [comment]
Thanks for resolving the merge conflicts and taking care of the other quick fixes @hafizrahman

**Pull request merged**